code/datums/status_effects/debuffs/slimed.dm

```diff
@@ -0,0 +1,69 @@
+/// The minimum amount of water stacks needed to start washing off the slime.
+#define MIN_WATER_STACKS 5
+/// The minimum amount of health a mob has to have before the status effect is removed.
+#define MIN_HEALTH 10
+
+/atom/movable/screen/alert/status_effect/slimed
+	name = "Covered in Slime"
+	desc = "You are covered in slime and it's eating away at you! Find a way to wash it off!"
+	icon_state = "slimed"
+
+/datum/status_effect/slimed
+	id = "slimed"
+	tick_interval = 3 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/slimed
+	remove_on_fullheal = TRUE
+
+	/// The amount of slime stacks that were applied, reduced by showering yourself in water.
+	var/slime_stacks = 10 // ~10 seconds of standing under a shower
+
+/datum/status_effect/slimed/on_apply()
+	if(owner.get_organic_health() <= MIN_HEALTH)
+		return FALSE
+	owner.visible_message(
+		span_warning("[owner] has been covered in a thick layer of slime!"),
+		span_userdanger("You have been covered in a thick layer of slime! Find a way to wash it off!"))
```

Comment:
Small thing but blind mobs won't get the latter message (`Find a way to wash it off!`) So if it's intended to be a hint it should be in a to chat